### DISCUSSION TITLE
Error: No mime type for file with extension gifsupported

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [facebook/docusaurus#8043] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/15354455/188176619-80638051-964a-43d8-86c6-ffa44ca76a83.png)
![image](https://user-images.githubusercontent.com/15354455/188177576-3fe4f7fd-1f8f-4ce0-af94-b699c9b9900d.png)

I am packing now and I have been reporting this mistake, and it has been unable to pass
dep: 
@babel/helper-string-parser": "^7.18.10",
  "@docusaurus-plugin/less": "^0.1.1",
  "@docusaurus/core": "^2.0.0-beta.14",
  "@docusaurus/plugin-google-gtag": "^2.0.0-beta.14",
  "@docusaurus/plugin-ideal-image": "2.0.0-beta.14",
  "@docusaurus/preset-classic": "^2.0.0-beta.14",

erros: 
Error: No mime type for file with extension gifsupported
Error: No mime type for file with extension gifsupported
Error: No mime type for file with extension gifsupported
Error: No mime type for file with extension gifsupported
[ERROR] Client bundle compiled with errors therefore further build is impossible.

# The problem above is  caused by @docusaurus/plugin-ideal-image 

But packing is still slow：

![image](https://user-images.githubusercontent.com/15354455/188266633-ea81567e-7c1f-4468-9126-f7361ab752d3.png)

**Comment by slorber** [comment, marked as answer]
This plugin doesn't support .gif yet

Duplicate of https://github.com/facebook/docusaurus/issues/4115
